software/docking.yml

```diff
@@ -0,0 +1,16 @@
+name: DocKing
+website_url: https://docking.shipsaas.tech
+description: Your shared-microservice that takes over the document templates management & render/export PDF dynamically.
+licenses:
+  - MIT
+platforms:
+  - PHP
+  - Javascript
+  - Docker
+tags:
+  - Document Management
+source_code_url: https://github.com/shipsaas/docking
+demo_url: https://docking-demo.shipsaas.tech/console
+stargazers_count: 69
+updated_at: '2023-10-29'
+archived: false
```

Comment:
This is all autofilled so we can get rid of it for now.

```suggestion
```

Reply:
cool, good to know, removed